shopinvader_wishlist/services/wishlist.py

```diff
@@ -113,6 +113,18 @@ def replace_items(self, _id, **params):
         self._replace_items(record, params)
         return self._to_json_one(record, **params)
 
+    @property
+    def access_info(self):
+        with self.shopinvader_backend.work_on(
+            "res.partner",
+            partner=self.partner,
```

Comment:
The naming of these properties is really confusing. What's the difference between partner, partner_user, invader_partner_invader_partner_user. It's strange to read that a partner_user is an instance of `res.partner` and not a `res.users`